lumigator/backend/backend/services/workflows.py

```diff
@@ -126,7 +126,7 @@ async def _run_inference_eval_pipeline(
 
         # wait for the evaluation job to complete
         status = await self._job_service.wait_for_job_complete(evaluation_job.id, max_wait_time_sec=60 * 10)
-        self._job_service._validate_results(evaluation_job.id, self._dataset_service.s3_filesystem)
```

Comment:
Not sure how this worked because _validate_results expects three args, not two...

Reply:
AFAICT in the base code it was called with 2 args... Maybe it didn't get checked in at some point?